client/src/components/FieldHelper.js

```diff
@@ -79,20 +76,18 @@ const Field = ({
       ) : (
         <InputGroup>
           {widgetElem}
-          {extraAddon && <InputGroup.Addon>{extraAddon}</InputGroup.Addon>}
           <FieldAddon id={id} addon={addon} />
         </InputGroup>
       ),
-    [addon, extraAddon, id, widgetElem]
+    [addon, id, widgetElem]
   )
   const validationState = getFormGroupValidationState(field, form)
   if (label === undefined) {
     label = utils.sentenceCase(field.name) // name is a required prop of field
   }
 
-  // setting label or extraColElem explicitly to null will completely remove these columns!
-  const widgetWidth =
-    12 - (label === null ? 0 : 2) - (extraColElem === null ? 0 : 3)
+  // setting label explicitly to null will completely remove this column!
```

Comment:
With this comment do we mean `// setting label explicitly to null will completely remove label column!` ?  `this column` doesn't refer to anything as far as i can see

Reply:
Note that particular functionality is unaffected by this PR. However, to clarify:
• label can be a string, in which case the given label will be shown for the input, taking up 2 spaces in the bootstrap grid
• label can be the empty string `""`, in which case an empty label will be shown, still taking up 2 spaces in the bootstrap grid
• label can be `undefined`, in which case a default label based on the field name will be shown, taking up 2 spaces in the bootstrap grid
• label can be `null`, in which case the whole label column will be removed from the bootstrap grid

Reply:
I meant, the `this column` in that comment doesn't have any reference to a previously defined column. For the writer of this code, what `this column` means may be apparent , but as a reviewer i had to look at the jsx and dive a bit more than necessary to understand what `this column` means. It would be clearer to explain which column is getting removed.

Reply:
Changed _this column_ to _the label column_.